php-checks/src/main/resources/org/sonar/l10n/php/rules/php/S1314.html

```diff
@@ -0,0 +1,25 @@
+<p>Integer literals starting with a zero are octal rather than decimal values. While using octal values is fully supported, most developers do not
+have experience with them. They may not recognize octal values as such, mistaking them instead for decimal values.</p>
+<h2>Noncompliant Code Example</h2>
+<pre>
+$myNumber = 023; # Noncompliant myNumber will hold 19, not 23 - was this really expected?
```

Comment:
I didn't know (or forgot) that `#` was a comment delimiter in PHP...
Good job for having created the PHP-specific RSPEC!